### ISSUE TITLE
Make sure all examples work with all python versions

### ISSUE THREAD

**Issue by hobarrera** [cocagne/txdbus#72]
Maybe this could be done via CI, just in case any API changes, etc (I guess we'd have to make the examples a package?).

**Comment by cocagne** [comment]
Hmmm. That's a good idea. Though I wonder if we could just create unit
tests that simply execute those examples and report pass/fail.

**Comment by hobarrera** [comment]
Yup, that would be the idea, but we need some extra work in most cases (eg: the one that fires a signals needs for us to start listening for the signal too).

I'd start out by moving `docs/examples/` into `/examples`, and create `examples/__init__.py`, though, so they can actually be importable.
